### ISSUE TITLE
The "check text spacing" option does not appear to set the paragraph spacing

### ISSUE THREAD

**Issue by sasgrw** [ThePacielloGroup/ARC-Tracker#7]
### Summary

When inspecting the CSS that is injected by selecting the "Check text spacing (WCAG 2.1)" option, the CSS only contains three of the four requirements by 1.4.12.

-    Line height (line spacing) to at least 1.5 times the font size;
-    **Spacing following paragraphs to at least 2 times the font size;**
-    Letter spacing (tracking) to at least 0.12 times the font size;
-    Word spacing to at least 0.16 times the font size.

```
<style type="text/css" id="ia11yVarsTextSpacingCss">
 *{line-height: 1.5 !important; 
   letter-spacing: 0.12em !important; 
   word-spacing: 0.16em !important;}
 </style>
```
_(Note, format of block above was adjusted to make it easier to read but it was copy/pasted from the Chrome devtools inspector)_

### Expected result

I expected to see a 4th CSS property such as `p{ margin-bottom: 2em !important; }` to match 1.4.12's second bullet point.

### Actual result

Three of the four requirements are properly applied.  The fourth requirement is missing.

### Test Case

Bring up any website and ARC Toolkit and check the box for "Check text spacing" then inspect the <head> element of the page.  The last element in the <head> block is a <style> that was injected by the ARC Toolkit.  It only contains three of the four CSS properties needed to test 1.4.12.

#### ARC Toolkit version

3.3.2.0

#### Operating System and version

Windows 10

#### Browser and version:

Chrome Version 84.0.4147.125 (Official Build) (64-bit)
